### PR TITLE
feat: app submission disclaimer

### PR DESCRIPTION
**Pull request by cliu02**
# Pull Request Template

## Issue Overview

This PR addresses #469. 

- [X] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This PR updates the copy for the app submission disclaimer, which is the last page for an application. 

## How Can This Be Tested/Reviewed?

Fill out a test application until you reach the "Terms" page. Then, confirm that the copy matches the text in the issue / copy doc. 

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by ColinBuyck** [comment]
@cade-exygy Did you leave the one review label by accident or do you think this needs another set of eyes? I trust your judgement here! Also @cliu02, just flagging the terms merge conflicts that have come up!

File: sites/public/src/pages/applications/review/terms.tsx [hunk 1]
```diff
@@ -94,32 +93,6 @@ const ApplicationTerms = () => {
...
   }
 
-  const agreeField = [
-    {
-      id: "agree",
-      label: t("application.review.terms.confirmCheckboxText"),
-    },
-  ]
-
-  const content = useMemo(() => {
```
Comment (emilyjablonski) [inline sites/public/src/pages/applications/review/terms.tsx:104]: Did we confirm with product that it's okay to remove the listing-type-specific text?

**Comment by emilyjablonski** [comment]
Calling out that the issue indicates `We need to store a record of their acknowledgement, if we don't already.` - have you checked in w product about not including that?

**Comment by YazeedLoonat** [comment]
Hey @cliu02 sorry I might not be understanding what you're looking for and maybe we can chat in a huddle or something 

when we store an application after its been submitted, we do link the user and the application

we do also store the timestamp for when a user has confirmed their account, and the createdAt timestamp on the application but those are stored in separate tables

**Review by emilyjablonski — APPROVED** [review]

**Pull request merged**